src/main/java/org/spongepowered/api/entity/living/Human.java

```diff
@@ -35,58 +35,85 @@
+     * <p>When exhaustion level reaches 0, saturation may begin to diminish.
+     * </p>
      *
-     * @return The current hunger
+     * @return The current exhaustion level
      */
-    float getHunger();
+    float getExhaustion();
 
     /**
-     * Sets the hunger value of this human entity.
-     * <p>Hunger is a type of exhaustion such that when the exhaustion
-     * depletes, saturation may begin to diminish.</p>
+     * Sets the exhaustion value of this human entity.
+     * <p>When exhaustion level reaches 0, saturation may begin to diminish.
+     * </p>
      *
-     * @param hunger The hunger level
+     * @param exhaustion The new exhaustion level
      */
-    void setHunger(float hunger);
+    void setExhaustion(float exhaustion);
 
     /**
-     * Gets the current food saturation of this human entity.
-     * <p>Saturation has health effects, depending on game difficulty and
-     * hunger levels. If the saturation is high enough, the human entity
-     * may heal. If the saturation is at 0, the human entity may starve.</p>
+     * Gets the current saturation level of this human entity.
+     * <p>Saturation acts as a buffer for food level. Food level will not
+     * decrease while the saturation level is greater than zero.</p>
      *
      * @return The current saturation level
      */
     float getSaturation();
 
     /**
-     * Sets the current food saturation of this human entity.
-     * <p>Saturation has health effects, depending on game difficulty and
-     * hunger levels. If the saturation is high enough, the human entity
-     * may heal. If the saturation is at 0, the human entity may starve.</p>
+     * Sets the saturation level of this human entity.
+     * <p>Saturation acts as a buffer for food level. Food level will not
+     * decrease while the saturation level is greater than zero.</p>
      *
-     * @param saturation The saturation level to set
+     * @param saturation The new saturation level
      */
     void setSaturation(float saturation);
 
     /**
+     * Gets the current food saturation of this human entity.
+     * <p>Food level has health effects, depending on game difficulty and
+     * hunger levels. If the food level is high enough, the human entity
+     * may heal. If the food level is at 0, the human entity may starve.</p>
+     *
+     * @return The current food level
+     */
+    float getFoodLevel();
+
+    /**
+     * Sets the current food saturation of this human entity.
+     *
+     * <p>Food level has health effects, depending on game difficulty and
+     * hunger levels. If the food level is high enough, the human entity
+     * may heal. If the food level is at 0, the human entity may starve.</p>
+     *
+     * @param foodLevel The new food level
+     */
+    void setFoodLevel(float foodLevel);
+
+    /**
      * Gets the current experience towards the next level.
      *
+     * <p>This value ranges from 0 to 1, with the experience bar
```

Comment:
I've based many of the methods in this PR off of Bukkit's spec, and this is how this method is defined. I can't imagine many use cases for this method (or more importantly, its setter) that don't depend on the percentage of the bar filled rather than the particular amount of experience, but I'm open to discussion about it.